api/admin/validator.py

```diff
@@ -163,3 +165,36 @@ def _value(self, field, form):
         elif len(value) == 1:
             return value[0]
         return filter(lambda x: x != None and x != "", value)
+
+
+class ValidatorFactory(object):
```

Comment:
This is specific to validators for patron authenticator protocols (as opposed to collection protocols or something else) and it should have a name that reflects that; maybe `PatronAuthenticationValidatorFactory`.

Reply:
Renamed `ValidatorFactory` to `PatronAuthenticationValidatorFactory`